sql/json-functions-generated-column.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Using TiDB as a Document Store
+category: user guide
+---
+
+# Using TiDB as a Document Store
+
+## About
+
+In order to functionally keep pace with the latest version of MySQL (version 5.7 and above) and provide better support for document storage business as well, TiDB supports JSON in the latest version. You can use JSON type fields in TiDB tables, and index the fields in the JSON document in the form of generated column. Based on this, you can flexibly deal with those business with uncertain schema, no longer limited by the bad read performance and lack of support for transaction in traditional document databases.
+
```

Comment:
To be compatible with MySQL 5.7 or later and better support the document store, TiDB supports JSON in the latest version. In TiDB, a document is a set of Key-Value pairs, encoded as a JSON object. You can use the JSON datatype in a TiDB table and create indexes for the JSON document fields using generated columns. In this way, you can flexibly deal with the business scenarios with uncertain schema and are no longer limited by the read performance and the lack of support for transactions in traditional document databases.